src/main/java/core/basesyntax/services/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.services;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private final String[] colors = new String[]{"red", "blue", "yellow", "green", "rose",
+        "black", "white", "violet", "orange", "indigo"};
+
+    public String getRandomColor() {
+        return colors[new Random().nextInt(colors.length - 1)];

```

Comment:
```suggestion
        return colors[new Random().nextInt(colors.length)];

```

java.util.Random.nextInt(int n) : The nextInt(int n) is used to get a random number between 0(inclusive) and the number passed in this argument(n), exclusive.